src/js/pages/jobs/JobDetailPage.js

```diff
@@ -97,7 +103,7 @@ class JobDetailPage extends mixin(StoreMixin, TabsMixin) {
       <button
         className="button button-inverse button-stroke"
         key="edit"
-        onClick={this.handleEditButtonClick.bind(this, jobID)}>
+        onClick={this.handleEditButtonClick.bind(this)}>
```

Comment:
I actually thought about refactoring this, but as this PR is only about introducing the edit modal, I decided not to adjust these. I'd prefer to do this in another PR, are you ok with that?